netgrasp/netgrasp.py

```diff
@@ -437,7 +437,8 @@ def log_event(mid, iid, did, rid, event, have_lock = False):
         ng.debugger.debug("entering log_event(%s, %s, %s, %s, %s, %s)", (mid, iid, did, rid, event, have_lock))
 
         # Only log events for which there are subscribers.
-        if ((ng.email["enabled"] and event in ng.email["alerts"]) or (ng.notification["enabled"] and event in ng.notification["alerts"])):
+        if (ng.email["enabled"] and event in ng.email["alerts"]) \
+            or (ng.notification["enabled"] and event in ng.notification["alerts"]):
```

Comment:
I really appreciate the kind feedback and explanations. It's a big help.

Glad to know you're ok with pep8. And yes, no worries, I'll start working with the 99 character limit.

So maybe we can create a "contributing" doc and start it with a "Project Style Guide" section? Don't feel rushed to make decisions. We can iterate on this.

So far so good with running netgrasp. I just got it running the other night and I've only been able to just briefly watch the output running it in the foreground. I wanted to use a custom config location alongside my local repo. I'm going to read more of the code to figure out how it handles the CLI. I also need to get a local mailbox setup.

Thanks again for the kind and thoughtful feedback, collaboration and accepting my PRs. :)